beacon-chain/rpc/apimiddleware/custom_hooks.go

```diff
@@ -265,13 +265,13 @@ func serializeV2State(response interface{}) (bool, []byte, gateway.ErrorJson) {
 }
 
 type phase0ProduceBlockResponseJson struct {
-	Version string                    `json:"version"`
-	Data    *beaconBlockContainerJson `json:"data"`
+	Version string           `json:"version"`
+	Data    *beaconBlockJson `json:"data"`
 }
 
 type altairProduceBlockResponseJson struct {
-	Version string                          `json:"version"`
-	Data    *beaconBlockAltairContainerJson `json:"data"`
+	Version string                 `json:"version"`
+	Data    *beaconBlockAltairJson `json:"data"`
```

Comment:
presumably this is addressing `Fixes wrong block structure for /eth/v2/validator/blocks`?

Reply:
Yes, this is correct. We don't need an additional wrapper, we should use `beaconBlockAltairJson` directly.